custom-backends/mssql/Startup.cs

```diff
@@ -95,8 +95,8 @@ LEFT JOIN
 
                 using (var cmd = new SqlCommand(sql, conn))
                 {
-                    cmd.Parameters.AddWithValue("@OrchestrationInstanceId", instanceId);
-                    cmd.Parameters.AddWithValue("@TaskHub", hubName);
+                    cmd.Parameters.Add(new SqlParameter("@OrchestrationInstanceId", SqlDbType.VarChar, 100) { Value = instanceId });
```

Comment:
I suggest to make it at least 256 symbols, since [it is the official limit for orchestrationIds](https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-orchestrations?tabs=csharp-inproc#orchestration-identity)

Reply:
Yes you are right juste used actual SQL size https://github.com/microsoft/durabletask-mssql/blob/b3a7bf4868d8cc19da1c91651293bb547b04aa43/src/DurableTask.SqlServer/Scripts/schema-1.0.0.sql#L17